packages/editor/src/components/inputs/Scrubber.tsx

```diff
@@ -37,18 +37,20 @@ import { useHookstate } from '@etherealengine/hyperflux'
 import Portal from '../layout/Portal'
 
 type ScrubberContainerProps = {
-  tag?: keyof JSX.IntrinsicElements
+  tag?: any
```

Comment:
why did we remove the type here? can we specify a type that is actually correct here instead of removing the typing

Reply:
i was getting two typescript errors otherwise 
Types of property 'ref' are incompatible. 
Type 'Ref<HTMLElement>' is not assignable to type 'LegacyRef<SVGSymbolElement> | undefined'
and
Expression produces a union type that is too complex to represent.ts(2590)

on line 48